pants

```diff
@@ -2,6 +2,32 @@
 # Copyright 2014 Pants project contributors (see CONTRIBUTORS.md).
 # Licensed under the Apache License, Version 2.0 (see LICENSE).
 
+function expected_python_version_to_run() {
+  if [[ "$PANTS_USE_PYTHON3" == 'true' ]]; then
+    echo '3'
+  else
+    echo '2'
+  fi
+}
+
+_PYTHON_VERSION_FILE="./.pants-python-version"
```

Comment:
We typically use `${REPO_ROOT}`.

However, when changing this line to `_PYTHON_VERSION_FILE="${REPO_ROOT}/.pants-python-version"`, I get the error `./pants: line 26: /.pants-python-version: Permission denied`, so I'm not sure what to do here.

Reply:
Well I think we need to source something (`common.sh`?) to get that `REPO_ROOT` variable?

Reply:
I will find the canonical method of doing this.

Reply:
Done! https://github.com/pantsbuild/pants/pull/7151/files#diff-ca425b88f047ce8ec45ee90e813ada91R5